xija/limits.py

```diff
@@ -0,0 +1,54 @@
+def get_limit_spec(limit):
+    """
+    Based on an input *limit* string of the form:
+
+    <system>.<type>.<direction>.<qualifier>
+
+    (where the "qualifier" element is optional) return the various pieces 
+    into a dictionary for convenience.
+    """
+    words = limit.split(".")
+    if len(words) < 3:
+        raise RuntimeError(f"{limit} is not a valid limit string")
+    limit_spec = {
+        "system": words[0],
+        "type": words[1],
+        "direction": words[2]
+    }
+    if len(words) == 4:
+        limit_spec["qualifier"] = words[3]
+    else:
+        limit_spec["qualifier"] = None
+    return limit_spec
+
+
+limit_colors = {
+    '.*.acisi': 'blue', 
```

Comment:
Should these be `*.*.acisi` to allow working with `get_limit_spec`?

Reply:
I don't think that this dict is meant to work with `get_limit_spec`.